### PR TITLE
make Mismatches system independent

### PR DESCRIPTION
**Pull request by kokodyn**
Fix for issue #209

### CODE REVIEW

**Comment by 0crat** [comment]
@kokodyn[/z](https://www.0crat.com/u/kokodyn) this pull request is too small, just 2 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @victornoel[/z](https://www.0crat.com/u/victornoel) please review this and merge or reject

File: src/main/java/org/llorllale/cactoos/matchers/Mismatches.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public Mismatches(
         this(
             args,
             new Joined(
-                new TextOf("\n"),
+                new TextOf(System.lineSeparator()),
```
[the remark this answers is not in the document]
Comment (kokodyn) [reply src/main/java/org/llorllale/cactoos/matchers/Mismatches.java]: @andreoss done

**Comment by victornoel** [comment]
@kokodyn @andreoss thx

**Comment by victornoel** [comment]
@rultor merge

**Pull request merged**